microprofile-reactive-messaging-kafka/README.adoc

```diff
@@ -998,7 +998,7 @@ The sub-sections describe how to deploy the application to OpenShift interacting
 
 NOTE: For the purposes of this Quickstart, you should choose one of these approaches, not both. Your choice should match the type of Kafka instance you selected earlier in <<installing-kafka-on-openshift, Installing Kafka on OpenShift>>.
 
-In both cases we will use Helm to install the application. Make sure you have downloaded the https://docs.openshift.com/container-platform/4.8/cli_reference/helm_cli/getting-started-with-helm-on-openshift-container-platform.html[Helm CLI Tool].
+In both cases we will use Helm to install the application. Make sure you have downloaded the https://docs.openshift.com/container-platform/4.9/applications/working_with_helm_charts/installing-helm.html[Helm CLI Tool].
```

Comment:
Can the `4.9` be made into a variable? Then we only need to change it in once place